save-frontend/src/main/kotlin/com/saveourtool/save/frontend/utils/ChildrenBuilderUtils.kt

```diff
@@ -104,3 +111,38 @@ fun ChildrenBuilder.selectorBuilder(
         }
     }
 }
+
+@Suppress("TOO_MANY_PARAMETERS", "LongParameterList", "LAMBDA_IS_NOT_LAST_PARAMETER")
+private fun ChildrenBuilder.buttonBuilder(
+    labelBuilder: ChildrenBuilder.() -> Unit,
+    style: String = "primary",
+    isDisabled: Boolean = false,
+    isOutline: Boolean = false,
+    isActive: Boolean = false,
+    classes: String = "",
+    title: String? = null,
+    onClickFun: MouseEventHandler<HTMLButtonElement>,
+) {
+    button {
+        type = ButtonType.button
+        val outline = if (isOutline) {
+            "outline-"
+        } else {
+            ""
+        }
+        val active = if (isActive) {
+            "active"
+        } else {
+            ""
+        }
+        className = ClassName("btn btn-$outline$style $active $classes")
+        disabled = isDisabled
+        onClick = onClickFun
+        title?.let {
+            asDynamic()["data-toggle"] = "tooltip"
+            asDynamic()["data-placement"] = "top"
+        }
+        this.title = title
```

Comment:
```suggestion
            this.title = title
        }
```
I think everything related to title should live in the same block